### PR TITLE
Arc-enabled Data Services AKS ADO Refresh

### PR DESCRIPTION
**Pull request by chintalavr**
Please review content and approve for merge

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md [hunk 1]
```diff
@@ -111,52 +111,47 @@ In this scenario, you will create a new Release pipeline to deploy the environme
...
 
   * Change the Template location to "URL of the file"
 
-  * Copy the raw URLs for both the [template](https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_data_jumpstart/aks/arm_template/mssql_mi/azuredeploy.json) and the [parameters](https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_data_jumpstart/aks/arm_template/mssql_mi/azuredeploy.parameters.json) json files and paste it in it's the proper field.
+  * Copy the raw URLs for both the [template](https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_data_jumpstart/aks/ARM/azuredeploy.json) and the [parameters](https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_data_jumpstart/aks/ARM/azuredeploy.parameters.json) json files and paste it in it's the proper field.
 
-  * The deployment ARM template requires you provide parameters values. Click on the _Edit Override template parameters_ to add your parameters values.
+  * The deployment ARM template requires you provide parameters values. Click on the *Edit Override template parameters* to add your parameters values.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md:120]:
```suggestion
  * The deployment ARM template requires you to provide parameters values. Click on the _Edit Override template parameters_ button to add the values of your parameters.
```

File: docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md [hunk 2]
```diff
@@ -111,52 +111,47 @@ In this scenario, you will create a new Release pipeline to deploy the environme
...
-  * *vmSize* - Client Windows VM size
-  * *resourceGroup* - Azure resource group where all the resources get deploy
-  * *AZDATA_USERNAME* - Azure Arc Data Controller admin username.  DO NOT USE 'sa' or 'admin'!!
-  * *AZDATA_PASSWORD* - Azure Arc Data Controller admin password (The password must be at least 8 characters long and contain characters from three of the following four sets: uppercase letters, lowercase letters, numbers, and symbols.)
-  * *ACCEPT_EULA* - "yes" **Do not change**
-  * *ARC_DC_NAME* - Azure Arc Data Controller name. The name must consist of lowercase alphanumeric characters or '-', and must start d end with a alphanumeric character (This name will be used for k8s namespace as well).
-  * *MSSQL_MI_NAME* - SQL Managed Instance name to be deployed on the Kubernetes cluster
-
+  * *githubAccount* - Target GitHub account
+  * *githubBranch* - Target GitHub branch
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md:140]: These are not needed as they are being used only for internal dev and test. By default, these are set to _microsoft_ and _main_ respectively. Any specific reason to incl. these?

File: docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md [hunk 3]
```diff
@@ -111,52 +111,47 @@ In this scenario, you will create a new Release pipeline to deploy the environme
...
+  
     > **Note: Make sure that you are using the same Azure resource group name as the one you've just used in the *azuredeploy.parameters.json* file**
 
     ![Screenshot of ARM template parameters config](./12.jpg)
 
     ![Screenshot of ARM template parameters config](./13.jpg)
 
-    ![Screenshot of ARM template parameters config](./14.jpg)
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md:151]: You have a file jump here. All images should be aligned and in the right order. i.e 13,14,15...

File: docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md (hunk 1 above)
Comment (likamrat) [reply docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md:120]: Update this also in the PG scenario

File: docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md (hunk 2 above)
Comment (likamrat) [reply docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md:140]: Also the same question on the PG scenario

File: docs/azure_arc_jumpstart/azure_arc_data/aks/aks_postgresql_hyperscale_arm_template_ado/_index.md [hunk 4]
```diff
@@ -152,11 +147,9 @@ In this scenario, you will create a new Release pipeline to deploy the environme
 
     ![Screenshot of ARM template parameters config](./14.jpg)
 
-    ![Screenshot of ARM template parameters config](./15.jpg)
-
 * Provide a deployment name.
 
-    ![Screenshot of deployment name config](./16.jpg)
+    ![Screenshot of ARM template parameters config](./15.jpg)
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_data/aks/aks_postgresql_hyperscale_arm_template_ado/_index.md:152]: You have a file jump here. All images should be aligned and in the right order. i.e 14,15,16...

**Review by likamrat — CHANGES_REQUESTED** [review]
Great work. Few fixes are needed

- SQL MI ADO scenario
  - Remove the use of github ARM params
  - Fix Image files jump
  - 02.jpg - Since this is the only button, no need for a red box and number. 
  - No need for this section anymore as it's being done via the automation
image
  - Change **ALL*  "Note" to "NOTE" 
image

- PG ADO scenario
  - Remove the use of github ARM params
  - Fix Image files jump
  - Remove old image 
image
  - 02.jpg - Since this is the only button, no need for a red box and number. 
  - No need for this section anymore as it's being done via the automation
image
  - Change **ALL*  "Note" to "NOTE" 
image

File: docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md (hunk 2 above)
Comment (chintalavr) [reply docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md]: Removed, no need to when we have default values.

File: docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md (hunk 3 above)
Comment (chintalavr) [reply docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md]: Updated

File: docs/azure_arc_jumpstart/azure_arc_data/aks/aks_postgresql_hyperscale_arm_template_ado/_index.md (hunk 4 above)
Comment (chintalavr) [reply docs/azure_arc_jumpstart/azure_arc_data/aks/aks_postgresql_hyperscale_arm_template_ado/_index.md]: Updated

**Review by likamrat — APPROVED** [review]
LGTM. Great work!

**Pull request merged**